mavisp/mavisp.py

```diff
@@ -235,6 +236,13 @@ def main():
     dataset_tables_path = out_path / 'dataset_tables'
     dataset_tables_path.mkdir(exist_ok=True)
 
+    # Generate a csv file that contains the number of mutations and the date of the run
+    time = strftime("%Y-%m-%d", gmtime())
+    nb_mutations = len(mfs.dataset_table['mutations'].explode().unique())
+    mutation_table = pd.DataFrame({'Number of mutations': nb_mutations, 'Date of run': time}, index=[0])
```

Comment:
would be good if you could also add:
* how many proteins we have (in general)
* how many proteins we have in just ensemble mode
* how many proteins we have in just simple mode
* how many proteins we have with both simple and ensemble mode